### PR TITLE
Port web_translate_dialog to 8.0

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Comment by dreispt** [comment]
Can you extract the description to a README.rst?
I tested this and my translation popup shows no lines to translate. Any clue?

**Comment by alexis-via** [comment]
@dreispt That's very strange ; it works well here at my end. You mean, when you open the translation popup on a product, you don't have anything ? When I do that, I have a pop-name like this one:
http://people.via.ecp.fr/~alexis/web_translate_dialog.png

Did you reload the interface after the installation of the module ?

I'll take care of README.rst when I have time.

**Comment by dreispt** [comment]
@alexis-via I see: this is for the translatable fields, not for the technical translations. Could you please make that clearer when extracting the README from the manifest?

File: web_translate_dialog/README.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Web Translate Dialog
+====================
+
+This module replaces the standard translation view by an easy-to-use pop-up view where you can translate all the fields of the object in all the installed languages (for the old users of OpenERP, it may remind them the translation pop-up of OpenERP 6.1).
```
Comment (pedrobaeza) [inline web_translate_dialog/README.rst]: s/OpenERP (first one)/Odoo

**Comment by pedrobaeza** [comment]
Tried on runbot and it's working fine. Please add in the "Known issues / Roadmap" a point about only open the affected field when clicking the flag icon instead of opening all the translatable fields, or if you want to improve it directly :wink: 

:+1:

**Comment by dreispt** [comment]
:+1:

**Comment by alexis-via** [comment]
@pedrobaeza In fact, I prefer the current behavior that shows all the translatable fields ; so I don't want to put in the "known issues" that it should open only the affected field :)

**Comment by pedrobaeza** [comment]
But that confuses the user to which field he/she should translate. Would you block a modification by my part changing this?

For now, I'm merging this one.

**Comment by alexis-via** [comment]
@pedrobaeza No, I wouldn't ! It's just my opinion, I'm not the author of this module.

**Comment by pedrobaeza** [comment]
OK, I will let it as configurable via system parameter for satisfying all tastes :wink:

**Pull request merged**